contrib/mod_sftp/fxp.c

```diff
@@ -14010,7 +14015,7 @@ static int fxp_send_display_login_file(uint32_t channel_id) {
 int sftp_fxp_handle_packet(pool *p, void *ssh2, uint32_t channel_id,
     unsigned char *data, uint32_t datalen) {
   struct fxp_packet *fxp;
-  int have_cache, res;
+  int res;
```

Comment:
Rather than having this `have_cache` variable here, reset to "false" whenever we handle a new packet, we need to keep that state elsewhere, so that it persists properly _across_ multiple SFTP packets.